dots.ipynb

```diff
@@ -0,0 +1,637 @@
+{
```

Comment:
<p>We first call dots with default parameters (i.e. similar to HiCCUPs). Later we then explore the various parameters than can be easily modified in cooltools.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open2c/open2c_examples/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>